packages/gasket-plugin-config/lib/middleware.js

```diff
@@ -16,6 +16,17 @@ function handler(gasket) {
         req,
         res);
 
+      // Error handling
+      // Better error description for the appRequestConfig & appEnvConfig lifecycles
+      // In the event a config object isn't returned from the hook(s)
+      // it will result in a error in the destructuring of req.config
+      if (!req.config) {
+        throw new Error(
+          // eslint-disable-next-line max-len
+          'req.config is undefined - Ensure that all plugins hooking into the appRequestConfig and appEnvConfig lifecycle return a config object.'
```

Comment:
👍🏻 Let's make the error message concise and specific. Can you do a check and throw for `appEnvConfig` [here](https://github.com/godaddy/gasket/blob/main/packages/gasket-plugin-config/lib/plugin.js#L12-L15).

You messaging could then be something like:
`An appRequestConfig lifecycle hook did not return a config object.`
and
`An appEnvConfig lifecycle hook did not return a config object.`